tests/utils/test_utils.py

```diff
@@ -69,3 +71,12 @@ def test_set_docstring_blank(self):
         self.assertRaises(ValueError, set_model_doc_string, onnx_model.doc_string, "sample")
         set_model_doc_string(onnx_model, "", True)
         self.assertEqual(onnx_model.doc_string, "")
+
+    def test_set_denotation(self):
+        this = os.path.dirname(__file__)
+        onnx_file = os.path.join(this, "models", "coreml_OneHotEncoder_BikeSharing.onnx")
+        onnx_model = load_model(onnx_file)
+        set_denotation(onnx_model, "1", "IMAGE", dimension_denotation=["DATA_FEATURE"])
+        if StrictVersion(onnx.__version__) >= StrictVersion('1.2.1'):
```

Comment:
using  @unittest.skipIf, instead of running an empty test on a low version onnx.